src/Quartz/IJobDetail.cs

```diff
@@ -65,6 +66,11 @@ public interface IJobDetail
         /// </summary>
         Type JobType { get; }
 
+        /// <summary>
```

Comment:
I think we should just break the API and change `JobType` to be of type `JobType`, maybe even change the property name to `Type`?